doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -680,6 +683,43 @@ Although some of these events may appear redundant, they cross important
 boundaries in the internal code (such as passing data across locks into new
 threads).
 
+.. _mclock-tblshoot-hdd-shard-config:
+
+Slow Requests or Slow Recovery With mClock Scheduler
+----------------------------------------------------
+
+.. note:: This troubleshooting is applicable only for HDD based clusters running
+   mClock scheduler and with the following OSD shard configuration:
+   ``osd_op_num_shards_hdd`` = 5 and ``osd_op_num_threads_per_shard_hdd`` = 1.
```

Comment:
so why include a detailed troubleshooting guide for the configuration that we've just moved from?

Reply:
I added this entry for users who are on older upstream releases to help resolve the issue if applicable and in case an upgrade isn't feasible immediately. The resolution is a configuration change and OSD restart which is probably quicker than performing the upgrade procedure.